app/api/routes.py

```diff
@@ -99,14 +98,7 @@ def apikey():
 
 # Helpers
 def get_resource(id):
-    resource = None
-    try:
```

Comment:
I think this is fine, and you're right about the results. This was probably due to copy pasting or occurred as we changed how we used the query api.